services/director-v2/src/simcore_service_director_v2/api/routes/dynamic_services.py

```diff
@@ -40,3 +51,45 @@ async def service_retrieve_data_on_ports(
     )
     # validate and return
     return RetrieveDataOutEnveloped.parse_obj(resp.json())
+
+
+@router.post(
+    "/{node_uuid}:start",
+    summary="start the dynamic-sidecar for this service",
+)
+async def start_dynamic_sidecar(
+    node_uuid: str,
+    start_dynamic_sidecar_model: StartDynamicSidecarModel,
```

Comment:
MINOR: sometimes having names so long is not only unnecessarily redundant but makes more difficult to read the code. I think it is the case for ``start_dynamic_sidecar_model``.

Reply:
Any better suggestion for the name?

Reply:
actually got renamed to `model` in this context but the type remains the same.